### DISCUSSION TITLE
About Sharpness, Rank, Artifacts and CNR metrics in 2020 challenge

### DISCUSSION THREAD

**Discussion by mclvnot** [facebookresearch/fastMRI#294] [Q&A] [answered]
Hello,

in Results of the 2020 fastMRI Challenge for Machine Learning MR Image Reconstruction, there  are extra evaluation metrics like rank, sharpness and artifacts. (All of us know CNR).  Which one  were used for sharpness (there are some sharpness metrics using gradients),  or  
 for rank  does it depend on radiologists votes or is it Wilcoxon signed rank?, or how could I evaluate my results with artifact metrics?

Best,

**Comment by mmuckley** [comment, marked as answer]
Hello @mclvnot, I assume you are referring to the results in Table IV.

The ranks are not based on actual metrics, but rather radiologist Likert score assessments of the quality of the images. For the radiologist eval, we used the following:

1=extremely sharp boundaries between areas
2= clear boundaries between areas with only minor blurring that is not bothersome
3= blurring at boundaries that decreases overall image quality
4 = unacceptable blurring for standard clinical imaging

**Comment by mclvnot** [comment]
Thank you Matthew,

But I actually want to ask the artifact and sharpness metrics.  According to your answer, for artifact and sharpness, ragiologists evaluate the examinations by means of likert score. But about Johnsons last RSNA article, they evaluate the reconstruction results in terms of sharpness and artifact,too. That's why, I thought there could be a standart metric for sharpness and artifact. (For sharpnes, average gradient magnitude etc. For artifact, I don't know).

Thank you,

**Reply by mmuckley** [reply]
@mclvnot I suspect in that paper they also used qualitative radiologist assessment. I think qualitative sharpness assessment is often done in clinical radiology papers. It's difficult to assess sharpness quantitatively (without Likert). There have been some attempts in this direction, such as [(Kim 2018)](https://ieeexplore.ieee.org/abstract/document/8363523) or [(Chan 2021)](https://onlinelibrary.wiley.com/doi/epdf/10.1002/mrm.28828), but these have not yet reached widespread adoption. Ultimately, the errors at high frequencies are heavily dominated by noise.

CC @pjohnson519 for the sharpness metrics in her paper in case I'm incorrect.

**Reply by pjohnson519** [reply]
That's correct Matt; artifacts and sharpness were evaluated with a qualitative radiologist assessment.

**Reply by mclvnot** [reply]
You mean There is no  quantitative evaluation for sharpness, artifact and etc.? I first think you could use average magnitude of gradients maybe. 
Thank you!

**Reply by mmuckley** [reply]
@mclvnot you could certainly try that technique. In a way that would be similar to the Fourier radial Error Spectrum plot of Kim 2018, but there may be other approaches as well. I don't know of any widely-accepted methods for this.

**Reply by mclvnot** [reply]
Noise in the high frquencies could dominate the quantitative results, so I have been trying to understand and implement your scheme or way.

Thank you again!

**Reply by mclvnot** [reply]
Hello again, 

I want to ask a question,too. In radiologists evaluation, how were images shown to the radiologists? With fully sampled  and reconstructed  slices  side by side? or just only reconstructed  slices which radiologists did not know anything about acceleration factor, reconstruction modality,  fully sampled version of it?

Thank you!

**Reply by mmuckley** [reply]
Hello @mclvnot, we used an online DICOM viewer and uploaded the images to a database for the radiologists to access. Radiologists had knowledge of which image was the ground truth and they could display the images however they wanted using the viewer. The viewer had most of the functionality that they work with as part of their clinical routine.

**Reply by mclvnot** [reply]
RadiAnt Dicom Viewer?  Yes, I heard it before.   I think you mean in radiologists assesment part, each radiologists could see both the reconstruction results and ground truth by knowing which they belongs to?   Firstly, I was doubtful about bias in evaluation because of seeing ground truth.  However, your way made me lightened up for my preperation. 

Thank you!

**Reply by mmuckley** [reply]
@mclvnot I don't think it was that one. I think there are quite a few companies that are offering online viewers now.

**Reply by mclvnot** [reply]
Ok, then. Thank you for your inform!
